docs/common/Setting up IDE.rst

```diff
@@ -0,0 +1,77 @@
+Setting up Netbeans IDE for Synfig
+==================================
+
+If you are a developer yourself, you might want to setup a development environment, so that you can debug and develop Synfig Studio yourself.
+This tutorial will guide you on setting up Apache Netbeans IDE for Synfig.
+It is recommended that you clone the Synfig project before following this guide :)
+
+Prerequisites
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Please ensure that you have the following prerequisites installed on your machine:
+
+    * Git
+    * Oracle’s Java 8 or Open JDK v8
+
+.. note::
+
+     **TL;DR**
+     You can also refer to this video tutorial -> https://youtu.be/op-PShrJTCM
+
+
+Installation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The Apache Netbeans IDE is recommended for Synfig Studio developments. The Installer automatically prepares the IDE so that it comes with all required plug-ins, the correct workspace encoding settings, pre-configured code formatters and more. Simply follow these steps:
+
+    * Download the `Netbeans Installer. <https://netbeans.apache.org/download/nb111/nb111.html>`_
```

Comment:
I think "https://netbeans.apache.org/download/" will be enought.

Reply:
done